pyfar/classes/coordinates.py

```diff
@@ -1155,8 +1155,8 @@ def z(self, value):
     @property
     def rho(self):
         r"""
-        Radial distance to the the z-axis of the right handed
-        Cartesian coordinate system in meters (0 \leq radius < \infty)."""
+        Distance perpendicular to the the z-axis of the right handed Cartesian
```

Comment:
```suggestion
        Radial distance to the the z-axis of the right handed Cartesian
```
This was correct before. A distance is not perpendicular, only the vector (with length corresponding to the distance) is perpendicular. ;) Please change back.